service/src/test/java/uk/gov/hmcts/reform/fpl/service/PreSubmissionTasksServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.enums.Event;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.submission.EventValidation;
+import uk.gov.hmcts.reform.fpl.service.validators.CaseSubmissionChecker;
+import uk.gov.hmcts.reform.fpl.service.validators.EventsChecker;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+class PreSubmissionTasksServiceTest {
+    @Mock
+    private EventsChecker eventsChecker;
+
+    @InjectMocks
```

Comment:
could call it testObject or underTest to eye catch who is the target of the test

Reply:
Done